controller/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.6.12-slim
+
+RUN set -ex \
+  ; apt-get update -yqq \
+  ; apt-get install -yqq git \
+  ; rm -rf /var/lib/apt/lists/* \
+  ; :
+
+RUN pip install --no-cache-dir kubernetes==12.0.1 git+https://github.com/sigopt/sigopt-python.git@sj/main
```

Comment:
@dwanderson-intel we might need to update the BOM because this installs all of the dependencies of the new sigopt-python client